From e338ced1e04bf4b97322d5eed2b5bdf5b052095a Mon Sep 17 00:00:00 2001
From: Krzysztof Konopko <kris@youview.com>
Date: Thu, 15 Sep 2016 13:01:49 +0200
Subject: [PATCH] shared/platform: include weston-egl-ext.h only if ENABLE_EGL
 is defined

Including `weston-egl-ext.h` causes compilation failure for configurations
with EGL disabled.

Verified with `--disable-egl`, `--disable-x11-compositor`
and `--disable-drm-compositor`.

Signed-off-by: Krzysztof Konopko <kris@youview.com>
Reviewed-by: Pekka Paalanen <pekka.paalanen@collabora.co.uk>
[yann.morin.1998@free.fr: backported from upstream]
Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>
---
 shared/platform.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/shared/platform.h b/shared/platform.h
index 1eb96fd..30db1a6 100644
--- a/shared/platform.h
+++ b/shared/platform.h
@@ -33,9 +33,9 @@
 #include <wayland-egl.h>
 #include <EGL/egl.h>
 #include <EGL/eglext.h>
-#endif
 
 #include "weston-egl-ext.h"
+#endif
 
 #ifdef  __cplusplus
 extern "C" {
-- 
2.7.4

